core/src/test/kotlin/org/evomaster/core/search/algorithms/MioAlgorithmOnTrackOneMaxTest.kt

```diff
@@ -0,0 +1,116 @@
+package org.evomaster.core.search.algorithms
+
+import com.google.inject.Injector
+import com.google.inject.Key
+import com.google.inject.Module
+import com.google.inject.TypeLiteral
+import com.netflix.governator.guice.LifecycleInjector
+import org.evomaster.core.BaseModule
+import org.evomaster.core.EMConfig
+import org.evomaster.core.search.algorithms.onemax.OneMaxIndividual
+import org.evomaster.core.search.algorithms.onemax.OneMaxModule
+import org.evomaster.core.search.algorithms.onemax.OneMaxSampler
+import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.mutator.StandardMutator
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.lang.IllegalArgumentException
+
+class MioAlgorithmOnTrackOneMaxTest {
+
+
+    private lateinit var config: EMConfig
+    private lateinit var mio: MioAlgorithm<OneMaxIndividual>
+
+
+    @BeforeEach
+    fun init(){
+
+        val injector: Injector = LifecycleInjector.builder()
+                .withModules(* arrayOf<Module>(OneMaxModule(), BaseModule()))
+                .build().createInjector()
+
+        mio = injector.getInstance(Key.get(
+                object : TypeLiteral<MioAlgorithm<OneMaxIndividual>>() {}))
+
+        config = injector.getInstance(EMConfig::class.java)
+
+        config.stoppingCriterion = EMConfig.StoppingCriterion.FITNESS_EVALUATIONS
+
+
+        val randomness = injector.getInstance(Randomness::class.java)
+        randomness.updateSeed(42)
+
+        val sampler = injector.getInstance(OneMaxSampler::class.java)
+        val n = 20
+        sampler.n = n
+
+    }
+
+
+    @Test
+    fun testIndividualWithTrack(){
+
+        config.enableTrackIndividual = true
+        config.enableTrackEvaluatedIndividual = false
+
+        val solution = mio.search()
+
+        solution.individuals.forEach { s->
+            Assertions.assertNull(s.getTrack())
```

Comment:
you do not need to repeat "Assertions." in each call. You can just do a static import of those static methods

Reply:
thanks. modified